ja/orm/database-basics.rst

```diff
@@ -364,17 +364,55 @@ timestamp か 整形した日付文字列に変換します。
 同様に 'binary' という名前の項目ならファイルハンドラを受け入れ、データを読み込むときには
 ファイルハンドラを生成します。
 
-.. versionchanged:: 3.3.0
-    ``json`` 型が追加されました。
+.. _datetime-type:
 
-.. versionchanged:: 3.5.0
-    ``smallinteger`` 型と ``tinyinteger`` 型が追加されました。
+DateTime タイプ
+--------------
```

Comment:
Apparently the underline is too short, at least the build warns about it. Anybody knows how exactly the chars are being counted?

Reply:
I don't know. These japanese devs know though.

Reply:
I'm pretty sure sphinx counts counts Kanji and Hirigana as 2 characters each.